src/Beacon.jsx

```diff
@@ -60,7 +68,8 @@ export class Beacon extends Component {
       tooltipHeight: 0,
       tooltipWidth: 0,
       tooltipHidden: false,
-      tooltipAttachment: null,
+      tooltipAttachmentVertical: null,
+      tooltipAttachmentHorizontal: null,
       inactive: false,
```

Comment:
Split the attachment vars so I can work with them easier without parsing the whole string.